Makefile

```diff
@@ -10,15 +10,15 @@ ETHERSCAN_API_KEY?=VEJ7GISNRKFUESRPC4W4D3ZEM2P9B4J6C4
 include .env
 
 deploy-arb-l1:
-	@set -e; \
-	echo "Deploying ArbitrumVerifier..." && \
```

Comment:
Why to do this way? I see that the result is the same as we are using @set e. It is for code legibility maybe?

Reply:
yep